src/traces/cone/index.js

```diff
@@ -22,7 +22,16 @@ module.exports = {
 
     meta: {
         description: [
-            '...'
+            'Use cone traces to visualize vector fields.',
+            '',
+            'Specify a vector field using 6 1D arrays,',
+            '3 position arrays `x`, `y` and `z`',
+            'and 3 vector component arrays `u`, `v`, `w`.',
+            '',
+            'If nothing else is provided, the cones are drawn exactly at the positions given',
+            'by `x`, `y` and `z`.',
+            'To specify the cone positions manually, use arrays attributes',
+            '`cones.x`, `cones.y` and `cones.z`.'
```

Comment:
"To display cones at different positions from the field data, use array attributes `cones.x`, `cones.y` and `cones.z`. Where these positions do not match data points, the field is interpolated ..." (How, what algorithm? And does the algorithm extend outside the convex hull of the data? Perhaps also mention interpolation in `attributes.cones.description`?)

Reply:
> How, what algorithm?

Can @kig help us out here?